### PR TITLE
ci(sonarcloud): configure sonarcloud scan

### PR DESCRIPTION
**Pull request by grubyak**
- rename steps
- provide `sonarcloud` integration

### CODE REVIEW

**Review by CezCz — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,27 +3,41 @@ on: pull_request
 
 jobs:
   test:
-    name: build & test
+    name: Build & test
```
Comment (rachitkumar13) [inline .github/workflows/ci.yml:6]: Maybe -> `Build & Test`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -3,27 +3,41 @@ on: pull_request
...
-      - name: Code coverage
+      - name: Provide code coverage info as comment in pull request
         uses: codecov/codecov-action@v1
+      - name: Provide sonar static code analysis as comment in pull request
+        uses: sonarsource/sonarcloud-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          projectBaseDir: .
```
Comment (rachitkumar13) [inline .github/workflows/ci.yml:39]: Do we need to provide it as `.` because I remember its by default.

Comment (grubyak) [reply .github/workflows/ci.yml:39]: yes, it's needed to provide `projectBaseDir` if other parameters are provided

File: .github/workflows/ci.yml (hunk 1 above)
Comment (grubyak) [reply .github/workflows/ci.yml]: done

File: .github/workflows/ci.yml (hunk 2 above)
Comment (rachitkumar13) [reply .github/workflows/ci.yml:39]: oks.

**Review by rachitkumar13 — APPROVED** [review]
LGTM!

**Pull request merged**